pkg/metadata/README.md

```diff
@@ -0,0 +1,3 @@
+# Metric Metadata
+
+This module is experimental and its functionality and interfaces are not guaranteed to be stable or permanent.
```

Comment:
Please also add this warning in doc comments for MetadataExporter, MetadataDelta, MetadataUpdate structs.